lib/cext/ruby.h

```diff
@@ -218,6 +218,7 @@ void rb_check_safe_obj(VALUE object);
 bool SYMBOL_P(VALUE value);
 
 // Constants
+#define RUBY_VM 1
```

Comment:
This is a lame check, they should test the version.
But let's define it and see if it causes problems.
FWIW it is defined in MRI as:
```c
#define RUBY_VM 1 /* YARV */
```